tests/test_util.py

```diff
@@ -30,7 +30,12 @@ def test_tokenize():
 def test_expand_parentheses():
     def xp(s): return {''.join(sent) for sent in expand_parentheses(tokenize(s))}
     assert xp('1 (2|3) 4 (5|6) 7') == {'12457', '12467', '13457', '13467'}
-    assert xp('1 (2 3) 4') == {'1(23)4'}
+    assert xp('1 (2 3) 4') == {'1234'}
```

Comment:
With Padatious, the goal is to make the language as close to natural language as possible. Since parentheses occur in everyday speech (like this), I don't think they should be removed if there's no `|` inside.

For example, `' '.join(expand_parentheses('I like apples ( the red ones )'.split(' '))[0])` turns into `'I like apples the red ones'`

Reply:
I have now amended this and the corresponding tests.
Yet I think that there should be another solution as this could confuse developers of skills and padatious alike. Maybe use some other kind of brackets for options, like `[]`? Otherwise it wouldn't seem obvious where brackets get lost or stay after parsing/expansion.